react/components/AccordionFilterItem.js

```diff
@@ -12,57 +13,50 @@ import { getFilterTitle, formatFacetToLinkPropsParam, HEADER_SCROLL_OFFSET } fro
 const AccordionFilterItem = ({
   title,
   options,
-  type,
   show,
   open,
   onOpen,
-  onItemSelected,
-  getLinkProps,
   intl,
+  filtersChecks,
+  handleFilterCheck,
 }) => (
   <Fragment>
-    <div
-      className={classNames('vtex-accordion-filter__item t-body pv3 ph7 pointer bb b--muted-4', {
+    {!open &&
+      <div  className={classNames('vtex-accordion-filter__item h3 t-body pv3 ph5 pv4 pointer bb b--muted-4', {
         'vtex-accordion-filter__item--active': open,
         'vtex-accordion-filter__item--hidden dn': !show,
       })}
       onClick={onOpen}
-    >
-      <div
-        className={classNames('vtex-accordion-filter__item-title', {
-          'c-on-base': open,
-        })}
       >
-        {getFilterTitle(title, intl)}
-
-        <span className="vtex-accordion-filter__item-icon fr">
-          <Arrow up={open} size={10} />
-        </span>
+        <div
+          className={classNames('vtex-accordion-filter__item-title pv4', {
+            'c-on-base': open,
+          })}
+        >
+          {getFilterTitle(title, intl)}
+          <span className="vtex-accordion-filter__item-icon fr">
+            <Arrow up={open} size={10} />
+          </span>
+        </div>
       </div>
-    </div>
+    }
+    
     {open && (
       <div className="vtex-accordion-filter__item-options">
         {options.map(opt => {
```

Comment:
Destructure using `{Name}` since you are only using this prop, and avoid repeating `opt.Name` all over the render